IdentityCore/src/cache/crypto/MSIDAssymetricKeyGeneratorFactory.m

```diff
@@ -40,15 +42,17 @@ @implementation MSIDAssymetricKeyGeneratorFactory
 
 + (id<MSIDAssymetricKeyGenerating>)iOSDefaultKeyGeneratorWithError:(NSError **)error
 {
-    return [[MSIDAssymetricKeyKeychainGenerator alloc] initWithGroup:nil error:error];
+    NSString *keychainGroup = [[MSIDKeychainTokenCache defaultKeychainCache] keychainGroup];
```

Comment:
I don't think default group is always the right one here

Reply:
Addressed. Created MSIDCacheConfig to address keychain group and SecAccessRef